code/game/objects/items/weapons/clown_items.dm

```diff
@@ -165,6 +168,37 @@
 					continue
 			M.emote("flip")
 
+/obj/item/weapon/bikehorn/saxophone
+	name = "saxophone"
+	desc = "NEVER GONNA DANCE AGAIN, GUILTY FEET HAVE GOT NO RHYTHM!"
+	icon = 'icons/obj/musician.dmi'
+	icon_state = "sax"
+	item_state = "sax"
+	force = 10
+	cooldowntime = 150
+	attack_verb = list("played blues on", "serenaded", "battered", "bashed")
+	honksound = list('sound/items/sax.ogg', 'sound/items/sax2.ogg','sound/items/sax3.ogg','sound/items/sax4.ogg','sound/items/sax5.ogg','sound/items/sax6.ogg')
+
+
+/obj/item/weapon/bikehorn/saxophone/attack_self(mob/user)
+	if(!spam_flag)
+		spam_flag = TRUE
+		playsound(src.loc, pick(honksound), 50, 0)

```

Comment:
Please use `get_turf(user)` here so we get the parent turf to play sounds on if you are inside a closet